raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,335 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: goerli
+  services:
+    pfs:
+      url: https://pfs-goerli.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_fund: 1_000_000_000_000_000_000
+
+nodes:
+  mode: managed
+  count: 5
+  raiden_version: local
+  enable-monitoring: false
+  default_options:
+    gas-price: fast
+    environment-type: development
+    default-settle-timeout: 40
+    default-reveal-timeout: 20
+    proportional-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    proportional-imbalance-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+
```

Comment:
Could add a description of what the scenario does like all other scenarios have? And also add it to the README?

Reply:
Will add it.